### ISSUE TITLE
Banner shows on top of navigation bar when instantiated manually

### ISSUE THREAD

**Issue by ndmeiri** [ask-fm/AFMInfoBanner#10]
Steps to repro:
- Construct a banner with custom background and text colors like so

``` Swift
let banner = AFMInfoBanner()
banner.customBackgroundColor = Colors.greenColor()
banner.customTextColor = UIColor.whiteColor()
banner.text = "Session confirmed!"
```
- Show the banner after your initial view controller appears.

``` Swift
override func viewDidAppear(animated: Bool) {
    super.viewDidAppear(animated)
    // Instantiate banner (see step 1) ...
    // Show the banner
    banner.show(true)
}
```

The banner appears on top of the navigation bar as shown in the screenshot below. But I believe the expected behavior should match the behavior of the [gif in the README](https://raw.githubusercontent.com/ask-fm/AFMInfoBanner/master/res/red_banner.gif).
![screenshot](https://cloud.githubusercontent.com/assets/8139148/12207366/0de26e86-b5fb-11e5-87f5-6ff3eefe7aae.png)

**Comment by crazyjooe** [comment]
It seems that your navigation bar is translucent - in this case we show banner on top of it. 
I tried reproducing your steps in our test project and got expected behaviour.
If you think that it's not the case, please share the whole example project where this is reproducible,  so that I can take a deeper look.

**Comment by ndmeiri** [comment]
@crazyjooe Hm, I tried printing `self.navigationController?.navigationBar.translucent` right before the banner is shown, but the result is `false`.

**Comment by ndmeiri** [comment]
There's currently no way to override that "show above a translucent navigation bar" behavior, right?

**Comment by crazyjooe** [comment]
@ndmeiri unfortunately no. Can you please provide a project where this issue can be reproduced? I didn't manage to repeat your scenario.

**Comment by RajanFernandez** [comment]
I have the same problem, but with both the instance and class methods for showing the banner. Any ideas on how to force the banner under the navigation bar?

**Comment by ndmeiri** [comment]
@RajanFernandez No, I don't have a workaround yet. @crazyjooe Unfortunately, I haven't had time to prepare a project for you that will reproduce the issue. @RajanFernandez, would you be willing to do so if you have a spare moment?

**Comment by RajanFernandez** [comment]
@ndmeiri I just made a sample project to reproduce the issue... but now it works fine for me. Must be something odd in my other project. Good luck solving the issue in your project.

**Comment by crazyjooe** [comment]
@RajanFernandez So it works now both in test and yours project?
I tried reproducing this issue several times with no luck. I'd really appreciate a reproducible test case for this.

**Comment by RajanFernandez** [comment]
@crazyjooe I haven't put it back in my main project yet, but it worked fine in the test project so there's no reason it shouldn't in the main one.

**Comment by vatsal1992** [comment]
i know that the question/issue was in **SWIFT** but i tried in **Objective C**. here it is..
to open banner in particular viewController.

```
- (void)showBanner:(NSString*)text{
    CGFloat spacing = 0.0f;
    UIView* targetView = self.view;
    UIView* aboveView = self.navigationController.navigationBar;
    AFMInfoBanner* banner_ = [[AFMInfoBanner alloc]initWithTargetView:targetView viewAboveBanner:aboveView additionalTopSpacing:spacing];
    [banner_ setStyle:AFMInfoBannerStyleError];
    [banner_ setText:text];
    [banner_ showAndHideAfter:2.0 animated:YES];

}
```
